micro-rdk/src/common/app_client.rs

```diff
@@ -187,12 +189,14 @@ impl AppClient {
     pub(crate) fn initiate_signaling(
         &self,
         rpc_host: String,
+        heartbeats_allowed: String,
```

Comment:
if we are going to hard-code it and opt out it not expected then we shouldn't need an argument for that